ui/src/app/pages/profile/profile-page.tsx

```diff
@@ -346,27 +338,30 @@ export const ProfilePage = withUserProfile()(class extends React.Component<
                   valueKey: 'verifiedInstitutionalAffiliation.institutionDisplayName',
                   disabled: true
                 })}
-                {!profile.verifiedInstitutionalAffiliation && <div style={{color: colors.danger}}>
-                  Institution cannot be empty. Please contact admin
-                </div>}
+                {requireInstitutionalVerification && !profile.verifiedInstitutionalAffiliation &&
+                  <div style={{color: colors.danger}}>
+                    Institution cannot be empty. Please contact admin.
```

Comment:
keeping the message here because it's a sign something has gone wrong if a real user sees it